scripts/compatibility-matrix-automation/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "compatibility-matrix-automation",
```

Comment:
It may be useful to model this project like `tools/bump-pack`. In that project, it's written in typescript but it's never actually compiled. Instead we run all of it through tsx.

Reply:
Great suggestion, thanks!